src/api/profiles.tsx

```diff
@@ -1,22 +1,31 @@
 import { handleEtagResponse, handleResponse } from "util/helpers";
 import type { LxdProfile } from "types/profile";
 import type { LxdApiResponse } from "types/apiResponse";
+import { withEntitlementsQuery } from "util/entitlements/api";
+
+export const profileEntitlements = ["can_view"];
```

Comment:
Profile API changes seems unrelated for the instance entitlements. We should not mix too many changes into one PR. This PR should focus on instance related changes.